src/Statistics/include/Statistics/DataSet.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef STATISTICS_DATA_SET_H_
+#define STATISTICS_DATA_SET_H_
+
+#include <cstdint>
+#include <optional>
+#include <vector>
+
+namespace orbit_statistics {
+
+class DataSet {
+  DataSet(const std::vector<uint64_t>* data, uint64_t min, uint64_t max)
+      : data_(data), min_(min), max_(max) {}
+  const std::vector<uint64_t>* data_;
+  uint64_t min_;
+  uint64_t max_;
+
+ public:
```

Comment:
We always put `public` first, then `private` at the bottom. The examples in the style guide do the same. So I'd ask to be consistent.